pkg/apis/metrics/types.go

```diff
@@ -27,3 +30,26 @@ type RawNode struct {
 type RawPod struct {
 	unversioned.TypeMeta `json:",inline"`
 }
+
+// Node-level metrics.
+type Node struct {
+	unversioned.TypeMeta `json:",inline"`
+	api.ObjectMeta       `json:"metadata,omitempty"`
+	Metrics              Metrics `json:"metrics,omitempty"`
+}
+
+// Pod-level metrics.
+type Pod struct {
+	unversioned.TypeMeta `json:",inline"`
+	api.ObjectMeta       `json:"metadata,omitempty"`
+	Metrics              Metrics `json:"metrics,omitempty"`
```

Comment:
Should this really be an optional field? What would be the usecase for a Pod without metrics? If it is optional, it should  be a pointer type.